src/main/java/com/uniqueAuction/web/product/request/ProductUpdateRequest.java

```diff
@@ -26,7 +26,7 @@ public class ProductUpdateRequest {
     private String stock;
 
 
-    public Product updateProduct(){
+    public Product updateToProduct(){
```

Comment:
convert 정도로 함수명을 해도 되지 않을까요?
어떤 객체로 바뀌는지는 return type을 보면 알 수 있습니다.

```suggestion
    public Product convert(){
```

써주신 함수명의 해석이 기능과 맞지않아 혼란을 주기도 합니다.
- save to Product : Product에 저장
- update to Product: Product 최신화

Reply:
이름계속고민중이었는데 수정하겠습니다